inference-engine/tests/functional/plugin/myriad/shared_tests_instances/single_layer_tests/logical.cpp

```diff
@@ -78,19 +81,19 @@ INSTANTIATE_TEST_CASE_P(smoke_EltwiseLogicalInt,
                                 ::testing::Values(ngraph::helpers::InputLayerType::PARAMETER),
                                 ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),
                                 ::testing::Values(CommonTestUtils::DEVICE_MYRIAD),
-                                ::testing::Values(Config{{InferenceEngine::MYRIAD_DETECT_NETWORK_BATCH, CONFIG_VALUE(NO)}})),
+                                ::testing::Values(additional_config)),
                         LogicalLayerTest::getTestCaseName);
 
 INSTANTIATE_TEST_CASE_P(smoke_EltwiseLogicalNotInt,
-                        LogicalLayerTestVPU,
+                        LogicalLayerTest,
```

Comment:
Can we rebase `smoke_EltwiseLogicalInt` on `LogicalLayerTest` test class too and totally remove `LogicalLayerTestVPU`?

Reply:
I don't think we should do this. Because this code is necessary for Myriad. Myriad plugin does not support `bool` and it is converted to `int` but at the same time nGraph requires `bool` for input.

Reply:
You define `LOGICAL_NOT` tests as `INSTANTIATE_TEST_CASE_P(smoke_EltwiseLogicalNotInt, LogicalLayerTest, ...)`
But `LOGICAL_AND` is still defined `INSTANTIATE_TEST_CASE_P(smoke_EltwiseLogicalInt, LogicalLayerTestVPU,...)`
I just don't see any difference between these cases